scripts/provisioning/s3_setup

```diff
@@ -362,6 +362,14 @@ then
         exit 1
     fi
 
+    echo "INFO: copying ssl certificate.."
+    cp -f /opt/seagate/cortx/s3/install/haproxy/ssl/s3.seagate.com.pem /etc/ssl/stx-s3/s3/
```

Comment:
pls check if this ssl filename is same as mentioned in "frontend s3-main" section of haproxy.cfg file.

Reply:
currently we have commented out the ssl file line in the haproxy.cfg file.

@bkirunge7 Please review this if s3.seagate.com.pem is the right ssl file name and it is being copied to right location.

Reply:
I have removed the ssl configuration section as of now, as we need more answers from Provisioner team.